.github/workflows/build.yml

```diff
@@ -12,6 +12,8 @@ jobs:
       with:
         python-version: 3.8
     - name: Install dependencies
-      run: python -m pip install flake8
+      run: python -m pip install -r requirements.txt
+    - name: Verify Jechová can be executed
```

Comment:
Hmm, this sounds like French Revolution 😬

Reply:
Není pí. Jechová nakonec [čarodějnice](https://github.com/python/miss-islington)?